lib/Service/ValueObject/ViewColumnInformation.php

```diff
@@ -57,7 +69,8 @@ public function offsetSet(mixed $offset, mixed $value): void {
 		if (!$this->offsetExists($offset)) {
 			return;
 		}
-		$this->data[(string)$offset] = $value;
+		$this->ensureType($offset, $value);
+		$this->data[$offset] = $value;
```

Comment:
It might be better to make ensureType return “value”.

`$this->data[$offset] = $this->ensureType($offset, $value);`